CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 
 * [CHANGE] Distributor: Reduce calculated `GOMAXPROCS` to be closer to the requested number of CPUs. #12150
 * [CHANGE] Query-scheduler: The query-scheduler is now a required component that is always used by queriers and query-frontends. #12187
+* [CHANGE] Rollout-operator: Add `watch` permission to the rollout-operators's cluster role. #12360
```

Comment:
[nit] Might be good to mention https://github.com/grafana/rollout-operator/pull/262 here for further context.